var/spack/repos/builtin/packages/py-pydeps/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyPydeps(PythonPackage):
+    """Python module dependency visualization."""
+
+    homepage = "https://pypi.python.org/project/pydeps"
+    url      = "https://pypi.io/packages/source/p/pydeps/pydeps-1.7.1.tar.gz"
+
+    version('1.9.0', sha256='ba9b8c7d72cb4dfd3f4dd6b8a250c240d15824850a415fd428f2660ed371361f')
+    version('1.7.1', sha256='7eeb8d0ec2713befe81dd0d15eac540e843b1daae13613df1c572528552d6340')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-enum34', type=('build', 'run'), when='^python@:3.3.99')
+    depends_on('py-stdlib-list', type=('build', 'run'))
+    depends_on('py-pyyaml', type=('build', 'run'))
```

Comment:
Where do you see this dependency listed?

Reply:
it's (optionally) imported in pydeps/py2depgraph.py, also in tests (I'll need to add the test dep flag):
```
coltrane (GIT/pydeps):pydeps --external ./pydeps
[
    "configparser", 
    "enum", 
    "stdlib_list", 
    "yaml"
]
['configparser', 'enum', 'stdlib_list', 'yaml']

```

Reply:
I've never used pydeps before, interesting.

Reply:
heh, I only found it after looking for something to do python dep sanity-checking in the context of spack packages specifically.

Reply:
Tests are not included in the tarball, only the git repo. So this should be good as-is.